config/crd/patches/cainjection_in_clusterfluentdconfigs.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
+  name: clusterfluentdconfigs.kubesphere.io
```

Comment:
`clusterfluentdconfigs.kubesphere.io` should be `clusterfluentdconfigs.fluentd.fluent.io`?